remoteit/desktop
# DESK-1133 websocket disconnect message

### Description


Fixing UI being reset when it was only supposed to be updated

### Test plan



1. Sign in
2. Go to setup your local system
3. Click add service or delete service
4. You should not see the "Websocket disconnected" message

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1133>
